### DISCUSSION TITLE
Request For Enhancement of 'Shopping List Update'

### DISCUSSION THREAD

**Discussion by ChipSkylark37** [mealie-recipes/mealie#4691] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I have alerting enabled, whenever someone adds to the 'Shopping List' to notify me via Pushover Notification Service. 

Right now, all it says is 

"Shopping List Updated
generic"

It would be helpful to have context added to this alert, such as "Milk added" or "Eggs checked off" or "Sugar removed"

### Please provide a concise description of the feature that would resolve your issue.

n/a

### Please consider and list out some caveats or tradeoffs made in your design decision

n/a

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by ChipSkylark37** [comment]
I will have to check through the settings again but even re-ordering the shopping list send me a notification for each one. Longer lists that I had to drop it, scroll up, then move it again I got multiple notifications.

**Reply by michael-genson** [reply]
If you use one of the custom notifications ([e.g. `json` or `jsons`](https://github.com/caronc/apprise/wiki/Notify_Custom_JSON)) it will include the list item ids as well

**Reply by ChipSkylark37** [reply]
Is this possible to get working with the notification service "Pushover" and custom json? 

Edit: I don't see option json data that can be added to Pushover, it looks like it's limited to including a URL and URL title:

https://github.com/caronc/apprise/wiki/Notify_pushover
